hazelcast-spring/src/main/java/com/hazelcast/spring/transaction/TransactionContextHolder.java

```diff
@@ -39,19 +48,65 @@ public boolean isTransactionActive() {
         return transactionActive;
     }
 
-    public TransactionContext getContext() {
-        return transactionContext;
+    public <K, V> TransactionalMap<K, V> getMap(String name) {
+        return transactionContext.getMap(name);
     }
 
-    /**
-     * @see TransactionContext#beginTransaction()
-     */
     public void beginTransaction() {
         transactionContext.beginTransaction();
         transactionActive = true;
     }
 
-    public void clear() {
-        transactionActive = false;
+    public void commitTransaction() throws TransactionException {
```

Comment:
Minor: maybe re-arrange the methods so that `commit`, `rollback` are grouped together and `get*` methods are separate, for easier readability.

Reply:
done